packages/web-components/fast-foundation/src/form-associated/form-associated.ts

```diff
@@ -444,8 +452,14 @@ export abstract class FormAssociated<
             if (typeof this.value === "string") {
                 this.proxy.value = this.value;
             }
+
+            this.proxy.setAttribute("slot", proxySlotName);
         }
 
+        this.proxySlot = document.createElement("slot");
+        this.proxySlot.setAttribute("name", proxySlotName);
```

Comment:
Should these two lines be inside the `if` block as well? I was looking at the detachProxy code below and wondering if there was a need to re-create the slot or not.